salt/config/__init__.py

```diff
@@ -2919,7 +2919,7 @@ def is_profile_configured(opts, provider, profile_name, vm_=None):
         required_keys.append('image')
         if driver == 'vmware':
             required_keys.append('datastore')
-    elif driver in ['linode', 'virtualbox', 'vmware']:
```

Comment:
That was part of the point of this PR--a VM created specifically from scratch can't use `clonefrom`, though it does need `image`.  I guess we need to check for this more robustly then?